src/objects.rs

```diff
@@ -704,6 +704,46 @@ pub struct Transfer {
     pub equipment_id: Option<String>,
 }
 
+#[derive(Serialize, Deserialize, Debug, Derivative, PartialEq)]
+#[derivative(Default)]
+pub enum TransportType {
+    #[derivative(Default)]
+    #[serde(rename = "0")]
+    Regular,
+    #[serde(rename = "1")]
+    ExclusiveSchool,
+    #[serde(rename = "2")]
+    Mixed,
```

Comment:
better names? @prhod @TeXitoi

Reply:
I'm not found of these names, but I can't find anything better now.

Reply:
For the `Mixed` value, it's meaning is not clear when read alone. What about `RegularAndSchool` ?

Reply:
ok for RegularAndSchool